### ISSUE TITLE
Shift+Ins pastes twice, or blocks input

### ISSUE THREAD

**Issue by WorMzy** [qutebrowser/qutebrowser#3420]
(may be related to https://github.com/qutebrowser/qutebrowser/issues/3419 )

Since I upgraded to webengine 5.10.0, I am experiencing unexpected behaviour when using Shift+Ins to paste text:

On KDE/plasma, the initial combination of Shift and Ins pastes the text, then any subsequent key pastes it again. (so pasting "text", then pressing 'a' results in "texttext" being entered, when "texta" would be expected)

On standalone openbox, the initial combination pastes the text, then any subsequent keypress is ignored. (so pasting "text", then pressing 'a' results in "text" being entered, when "texta" would be expected)

I can't reproduce this behaviour in other webengine browsers (tried qupzilla and konqueror), nor in the webkit backend for qutebrowser.

Initial paste generates the following in debug:
```
11:28:02 DEBUG    misc       utils:get_clipboard:796 Primary selection contained: 'text'
11:28:02 DEBUG    commands   command:run:485 command called: insert-text ['text']
11:28:02 DEBUG    commands   command:run:500 Calling qutebrowser.browser.commands.CommandDispatcher.insert_text(<qutebrowser.browser.commands.CommandDispatcher>, 'text')
11:28:02 DEBUG    webview    webenginetab:_js_cb_single:506 Got element from JS: {'attributes': {'autocapitalize': 'off', 'autocomplete': 'off', 'autocorrect': 'off', 'class': 'js-search-input search__input--adv', 'id': 'search_form_input_homepage', 'name': 'q', 'tabindex': '1', 'type': 'text', 'value': ''}, 'caret_position': 0, 'class_name': 'js-search-input search__input--adv', 'id': 20, 'outer_xml': '<input id="search_form_input_homepage" class="js-search-input search__input--adv" type="text" autocomplete="off" name="q" tabindex="1" value="" autocapitalize="off" autocorrect="off">', 'rects': [{'bottom': 358.984375, 'height': 45.953125, 'left': 138.203125, 'right': 616.640625, 'top': 313.03125, 'width': 478.4375}], 'tag_name': 'INPUT', 'text': '', 'value': ''}
11:28:02 DEBUG    webelem    webelem:is_editable:247 Checking if element is editable: <qutebrowser.browser.webengine.webengineelem.WebEngineElement html='<input id="search_form_input_homepage" class="js-search-input search__input--adv" type="text" autocomplete="off" name="q" tabindex="1" value="" autocapitalize="off" autocorrect="off">'>
11:28:02 DEBUG    webelem    webengineelem:insert_text:148 Inserting text into element <qutebrowser.browser.webengine.webengineelem.WebEngineElement html='<input id="search_form_input_homepage" class="js-search-input search__input--adv" type="text" autocomplete="off" name="q" tabindex="1" value="" autocapitalize="off" autocorrect="off">'>

```
Nothing shows up in debug for the subsequent keypress.

**Version info:**
qutebrowser v1.0.4
Git commit: 4ffdd9da0 (2017-12-19 08:57:33 +0100)
Backend: QtWebEngine (Chromium 61.0.3163.140)

CPython: 3.6.3
Qt: 5.10.0
PyQt: 5.9.2

sip: 4.19.6
colorama: 0.3.9
pypeg2: 2.15
jinja2: 2.10
pygments: 2.2.0
yaml: 3.12
cssutils: 1.0.2 $Id$
attr: 17.3.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: yes
pdf.js: no
sqlite: 3.21.0
QtNetwork SSL: OpenSSL 1.1.0g  2 Nov 2017

Style: Breeze::Style
Platform: Linux-4.14.6-1-ARCH-x86_64-with-arch, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.6/site-packages/qutebrowser
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt

Paths:
cache: /home/v1wtykas/.cache/qutebrowser
config: /home/v1wtykas/.config/qutebrowser
data: /home/v1wtykas/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser
system data: /usr/share/qutebrowser

Uptime: 0:00:00

**Comment by The-Compiler** [comment]
It looks like events blocked by qutebrowser are still somehow forwarded by Qt. When I do `:bind --mode insert <ctrl-a> message-info foo` for example, I get that "foo" message, but then with the next key I type, all text gets selected (as if the event was passed through).

**Comment by The-Compiler** [comment]
Actually, this indeed looks like a duplicate of #3419.
